petastorm/spark/spark_dataset_converter.py

```diff
@@ -517,6 +536,19 @@ def wait_for_file(path):
         pool.join()
 
 
+def _check_dataset_file_median_size(url_list):
```

Comment:
Where is the function being called from?

Reply:
Oh, forgot, will call it when dataframe materialized finished.